.github/workflows/build.yml

```
@@ -138,7 +138,7 @@ jobs:
           echo "::set-output name=version::$VERSION"
 
       - name: Semantic Release
-        uses: cycjimmy/semantic-release-action@767cc0b0cd610ffb1ffee1b36a9b39120adc5f80
+        uses: cycjimmy/semantic-release-action@v2
```

On va évité de faire ça pour des 3rd parties non trustées
 ```suggestion
        uses: cycjimmy/semantic-release-action@7f0c0e4904fb36f980686dda081f2f258500f3ea
```